README.md

```diff
@@ -88,6 +88,29 @@ while (transactionPage.nextPage) {
 }
 ```
 
+#### Transfers
+
+To create a transfer:
+
+```typescript
+const transfer = await client.models.transfer.create({
+  amount: <amount>,
+  recipient: <recipent_name>,
+  iban: <recipent_iban>,
+  note: <optional_description>,
+  e2eId: <optional_e2eId>,
+});
+```
+
+To confirm a transfer:
+
+```typescript
+const result = await client.models.transfer.confirm(
+  <transfer_id>,
```

Comment:
This has been recently renamed to `confirmation_id`.

Reply:
I don't see it on latest backend

Reply:
corrected in https://github.com/kontist/sdk/pull/10/commits/982f621b4a7286b7b394f9f6e5e6cfb653f14261